src/views/components/admin/parkingLotCard.js

```diff
@@ -0,0 +1,27 @@
+import React, {useState, useCallback} from 'react';
+import { Card, Button, Alert, Row, Col, Collapse } from 'react-bootstrap';
+import { FormatDateTime } from '../../utils/time.js';
+
+function ParkingLot({parkingLot, dataTestID}) {
+    const [open, setOpen] = useState(false);
+
+    return (
+        <>
+            <Card className="mb-3" data-test-id={`${dataTestID}-card`}> 
```

Comment:
This needs more fields to include all parking lot stats, see https://github.com/IgorSteps/easypark/blob/main/docs/API_SPEC.md#2-get-all-parking-lots-api-endpoint